lumps/fdm_deh.lmp

```diff
@@ -75,31 +75,31 @@ par 32 0
 STARTUP5============================================================================\nThis is FreeDM, the free content deathmatch FPS.\n\nFreeDM is freely redistributable under the terms of the modified BSD\nlicense. Check out the Freedoom website for more information:\n\n    https://freedoom.github.io/\n============================================================================
 GOTARMOR = picked up a light armor vest.
 GOTMEGA = picked up a heavy armor vest.
-GOTHTHBONUS = got a 1% health bonus.
-GOTARMBONUS = got a 1% armor bonus.
+GOTHTHBONUS = got a vitality boost.
+GOTARMBONUS = used a shield charge.
 GOTSTIM = picked up a small health pack.
 GOTMEDINEED = found a large health pack, just in time!
 GOTMEDIKIT = picked up a large health pack.
-GOTSUPER = you got the overdrive sphere!
+GOTSUPER = you got the vitality totem!
 GOTBLUECARD = picked up a blue passcard.
 GOTYELWCARD = picked up a yellow passcard.
 GOTREDCARD = picked up a red passcard.
 GOTBLUESKUL = picked up a blue skeleton key.
 GOTYELWSKUL = picked up a yellow skeleton key.
 GOTREDSKULL = picked up a red skeleton key.
-GOTINVUL = temporary invulnerability!
-GOTBERSERK = you got the steroid pack!
-GOTINVIS = you got the stealth sphere!
+GOTINVUL = you are untouchable!
+GOTBERSERK = punch 'em to death!
+GOTINVIS = you got the stealth totem!
```

Comment:
Why "totem"? Makes me think of a totem pole. I don't think this works in English.